### DISCUSSION TITLE
Posting comment on PR as user

### DISCUSSION THREAD

**Discussion by sergeh** [community/community#27075] [Actions]
<p>Is it possible to post a comment as a user instead of <strong>github-actions</strong> ?</p>

**Comment by jsoref** [comment, marked as answer]
<p>If you give the workflow a PAT from that user…<br>
<a href="https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token" class="onebox" target="_blank" rel="noopener nofollow ugc">https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token</a></p>

**Comment by sergeh** [comment]
<p>That worked perfectly, thanks</p>
